ocaml/xapi/xapi_globs.ml

```diff
@@ -1805,6 +1807,10 @@ module Resources = struct
     ; ("depmod", depmod, "Path to depmod command")
     ; ("dracut", dracut, "Path to dracut command")
     ; ("udevadm", udevadm, "Path to udevadm command")
+    ; ( "pvsproxy_close_cache_vdi"
```

Comment:
I'm not sure it can be essential with regard to xapi exclusively. I don't think xcp-ng has pvsproxy available

Reply:
You're right, it's in the wrong list.

Reply:
Fixed.